ComplementaryScripts/modelTests/essentialGenes.m

```diff
@@ -0,0 +1,145 @@
+function [accurancy,tp,tn,fn,fp] = essentialGenes
+% essentialGenes
+%   Modify media + find essential genes in model. Adapted from:
+%   https://doi.org/10.1371/journal.pcbi.1004530
+%
+%   accurancy   (tp+tn)/(tp+tn+fn+fp)
+%   tp          true positives
+%   tn          true negatives
+%   fn          false negatives
+%   fp          false positives
+%
+%   Usage: [accurancy,tp,tn,fn,fp] = essentialGenes
+%
+%   Feiran Li, 2019-09-25
+%   Benjamin J. Sanchez, 2020-01-01
+%
+
+initCobraToolbox
+cd ..
+model = loadYeastModel;
+cd modelTests
+ko_tol = 1e-6;
+
+modeltemp = ravenCobraWrapper(model);
+model.rxnGeneMat = modeltemp.rxnGeneMat;
```

Comment:
are these 2 lines necessary? I thought `loadYeastModel` already took care of properly building rxnGeneMat:

https://github.com/SysBioChalmers/yeast-GEM/blob/bf49eaafb997769e2fb5f4adff8b124fe6a93c8d/ComplementaryScripts/loadYeastModel.m#L13

Reply:
No, it is not necessary, please remove that. Thanks!

Reply:
done in [`327ff2d`](https://github.com/SysBioChalmers/yeast-GEM/pull/200/commits/327ff2d570cf664ae699641415d1c59abf2a5083)